### DISCUSSION TITLE
InternetTest v8.7.0.2411

### DISCUSSION THREAD

**Discussion by lpeyr** [Leo-Corporation/InternetTest#630] [Releases]
# What's new?
### Added

- 6ec1d41f9db71c5f5711ec68f283a1741249ab3b Added new Default Storage path (#622)
- 0bdb10fb5209cf20c5e0b99fcb8de048b1f7167e Added Quick Information in header section (#623)
- 1d9b358677bbc3cccf6e51a7bb6baef54540fba8 Added translations (#626)
- 546b5675e09ef0be257720935b2d0822d8aaf6da Added possibility to set the default zoom level for Maps providers (#626)
- 92f6688c3e2e8a194a6e01ec9d8add91526cf4dc Added default zoom support for Bing maps (#626)

### Fixed

- bec24e33f23435094391991caf7336b723440197 Fixed caret not visible in TextBoxes (#624)
- a31d77728a529e424e57abe199cd48cc514d6fd8 Fixed theme selector not working on First Run (#625)

### Updated

- fe241c4f2b5e44983f6acd870dc06fc0359f22d9 Updated RestSharp

<hr /><em>This discussion was created from the release <a href='https://github.com/Leo-Corporation/InternetTest/releases/tag/8.7.0.2411'>InternetTest v8.7.0.2411</a>.</em>

**Comment by paradoxicallist** [comment]
InternetTest.exe (portable) missing. Thanks.

**Reply by lpeyr** [reply]
Fixed!
